examples/jsm/loaders/NRRDLoader.js

```diff
@@ -472,7 +473,11 @@ class NRRDLoader extends Loader {
 
 		volume.inverseMatrix = new Matrix4();
 		volume.inverseMatrix.copy( volume.matrix ).invert();
-		volume.RASDimensions = new Vector3( volume.xLength, volume.yLength, volume.zLength ).applyMatrix4( volume.matrix ).round().toArray().map( Math.abs );
+		volume.RASDimensions = new Vector3( 
```

Comment:
Why do we need to create a vector, why not just array immediately?

Reply:
Yes, you are right. I have yet to notice it. Will update it soon. : )